crates/ruff_linter/src/rules/ruff/rules/useless_if_else.rs

```diff
@@ -11,18 +11,15 @@ use ruff_python_ast::comparable::ComparableExpr;
 /// Useless `if`-`else` conditions add unnecessary complexity to the code without
 /// providing any logical benefit.
 ///
-/// Assigning the value directly is clearer and more explicit, and
-/// should be preferred.
+/// Assigning the value directly is clearer.
```

Comment:
Probably doesn't really need the paragraph break either -- could just have this as a second sentence of the previous paragraph!